torch/_torch_docs.py

```diff
@@ -12705,6 +12705,9 @@ def merge_dicts(*dicts):
 :attr:`dx` can be used to specify a different constant spacing, and :attr:`x` can be
 used to specify arbitrary spacing along :attr:`dim`.
```

Comment:
I think its enough to just inline have the new `Only one of :attr:`x` or :attr:`dx` should be specified.` sentence here.  Rather than introduce a new Note section.